README.md

```diff
@@ -1 +1,32 @@
-## Battleship with Java
\ No newline at end of file
+## Battleship with Java
+
+ Battleship game built with  Java 11 and  Spring Boot 2.5.6
+
+## Installation
+1.Clone the repository to your computer.
+
+2.Import the app into your IDE through build.gradle file.
+
+## Start
+To start the application run the BattleshipApplication(./src/main/java/com.game.Battleship/BattleshipApplication)
```

Comment:
Is this supposed to be a command? If so, please put in a code block. If not, please put a space before the `(`

Reply:
It is not a common, it is the path to BattleshipApplication. I've changed a bit the wording.